crm_phonecall/wizard/crm_phonecall_to_phonecall.py

```diff
@@ -56,16 +56,17 @@ def action_schedule(self):
         phonecall_obj = self.env['crm.phonecall']
         phonecalls = phonecall_obj.browse(self._context.get('active_ids', []))
         for this in self:
-            phonecalls.schedule_another_phonecall(
+            new_phonecalls_dict = phonecalls.schedule_another_phonecall(
                 this.date,
                 this.name,
                 this.user_id.id,
                 this.team_id.id or False,
                 this.tag_ids.ids,
                 action=this.action,
             )
-
-        return phonecalls.redirect_phonecall_view()
+        new_phonecalls = reduce(lambda x, y: x + y,
+                                new_phonecalls_dict.values())
+        return new_phonecalls.redirect_phonecall_view()
```

Comment:
If I'm not wrong, `new_phonecalls` is now a `list`, which shouldn't have a `redirect_phonecall_view()` method.

Reply:
No, you are wrong :stuck_out_tongue_winking_eye: 

It returns a recordset. The reduce is for that.

Reply:
How does that happen? 👀

Reply:
Well, check reduce documentation

Reply:
OK this code was not quite clear to me, but now I understand. It wasn't the problem with `reduce` (I know how that works bro 😆), it's more about the actual implementation... well, read above comment.

In any case, I'd have made just `sum(new_phonecalls_dict.values())`

Reply:
`sum` works with recordsets? TIL then